fix(tests): create a task to ensure latest release, beta and esr builds are available

jrgm opened the pull request:
Hey @vladikoff,

I wrote a tool (https://github.com/jrgm/fxdownload/blob/master/index.js) that fetches the `latest`, `latest-beta`, and `latest-esr` from http://ftp.mozilla.org/pub/mozilla.org/firefox/releases. I then setup a job to kick off at 00:00 UTC to ensure we have up to date firefox images in `/home/ubuntu/firefox-channels/latest{,-beta,-esr}`.

This change is just to get the bash from the teamcity command-line job out of a text box and into git control.

Note: to actually start using latest release and beta versions of Firefox, we need to move to selenium server 2.45.0 on the tc server.

jrgm commented:
Example output of run (ignore the irrelevant `g_slice_set_config` chirps:
  http://tests.dev.lcip.org:8111/viewLog.html?buildId=1529&buildTypeId=fxa_RefreshFirefoxBuildImages&tab=buildLog#_focus=18&state=18

vladikoff commented:
This is cool! I am going to update selenium tomorrow!

jrgm commented:
@vladikoff bump

vladikoff commented on tests/teamcity/update-builds.sh:
```diff
@@ -0,0 +1,26 @@
+  (cd /home/ubuntu/fxdownload && git pull)
+fi
+
+cd /home/ubuntu/fxdownload && \
```
Yeap I think we can get rid of `&&`

vladikoff commented on tests/teamcity/update-builds.sh:
```diff
@@ -0,0 +1,26 @@
+set -o errexit # exit on first command with non-zero status
+set -o nounset
+
+mkdir -p /home/ubuntu/firefox-channels
```
can we make /home/ubuntu configurable?

vladikoff replied:
so that we can run it on ubuntu vms (or ubuntu installs that do not have an ubuntu user).
Could be useful to run the tests locally and get the needed firefox versions.

jrgm replied:
Good point. I'll make it a command line arg, then $HOME, then /home/ec2-user ;-)

jrgm commented:
I updated and squashed this. The locations for where channels are downloaded and where the tool is installed can now be configured on the command line.
